inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/configuration/Constants.java

```diff
@@ -280,6 +281,12 @@ public class Constants {
             .defaultValue(5)
             .withDescription("minutes");
 
+    public static final ConfigOption<String> METRICS_LABELS =
+            ConfigOptions.key("inlong.metric.label")
+                    .noDefaultValue()
+                    .withDescription("INLONG GROUP ID + '&' + STREAM ID + '&' + NODE ID");
+
+
```

Comment:
It is different with `INLONG_METRIC` of  `inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/Constants.java`